demoRefixation.py

```diff
@@ -0,0 +1,55 @@
+from EMRM import *
+import random
+import string
+
+launch_list = range(-10,0)
+wlen_list = range(3, 11)
+
+launch_fix_dur = 2
+lpos_fix_unit = 1
+max_dur = 60
+n_trials = 5
+
+alpha = 0.9
+beta = 0.7
+
+vocab_fn = "./data/example_refix_vocab.csv"
+output_fn = "./refixation_output.txt" 
+
+output = ";".join(["wlen", "lpos", "launch", "word", "refix_act"]) + "\n"
+for wlen in wlen_list:
+    vocab_wlen = Vocabulary(string.ascii_lowercase,
+                           WLEN = wlen,
+                           csv_fn = vocab_fn)
+    vocab_chr_mat = vocab_wlen.get_onehot_chr_rep()
+    
+    reader_wlen = OneVirtualReader(vocabulary = vocab_wlen,
+                               SIGMA_SCALE = 1,
+                               LAMBDA_SCALE = 1,
```

Comment:
argument names shouldn't be all caps like this. The usual convention is that something in all caps is a hardcoded constant.